resources/messages/errorMessages.json

```diff
@@ -0,0 +1,7 @@
+{
+	"DEVICES": {
```

Comment:
These are not constants in the code. Can we use pascalCase or CamelCase for the keys? Currently they make my eyes bleed.